mpas_analysis/config.default

```diff
@@ -1282,6 +1282,41 @@ variables = [{'name': 'temperature',
 # zmax = -400
 
 
+[timeSeriesTransport]
+## options related to plotting time series of transport through transects
+
+# list of ocean transects from geometric_features to plot or ['all'] for all
+# available transects.
+transectsToPlot = ['Drake Passage', 'Tasmania-Ant', 'Africa-Ant', 'Antilles Inflow',
+                   'Mona Passage', 'Windward Passage', 'Florida-Cuba', 'Florida-Bahamas',
+                   'Indonesian Throughflow', 'Agulhas', 'Mozambique Channel', 'Bering Strait',
+                   'Lancaster Sound', 'Fram Strait', 'Nares Strait']
```

Comment:
@milenaveneziani and @vanroekel, I am open to which transects are included by default.  These are the 15 from QuickViz (https://github.com/MPAS-Dev/MPAS-QuickViz/blob/master/ocean/transects/python/compute_transects.py) -- the Robeson Channel, mentioned on QuickViz, doesn't seem to be in `geometric_features`.

We might also want to limit ourselves to a subset (or a different set) of these is `polarRegions.conf`.

Reply:
The Robenson Channel used to be basically Nares Strait.. I removed it from the features and replaced it with 'Nares Strait'.  So, yes, we could remove it from this list.